extensions/cache/runtime/src/main/java/io/quarkus/cache/runtime/devconsole/CacheDevConsoleRecorder.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.cache.runtime.devconsole;
+
+import io.quarkus.cache.runtime.CaffeineCacheSupplier;
+import io.quarkus.cache.runtime.caffeine.CaffeineCache;
+import io.quarkus.devconsole.runtime.spi.DevConsolePostHandler;
+import io.quarkus.devconsole.runtime.spi.FlashScopeUtil.FlashMessageStatus;
+import io.quarkus.runtime.annotations.Recorder;
+import io.vertx.core.Handler;
+import io.vertx.core.MultiMap;
+import io.vertx.ext.web.RoutingContext;
+
+@Recorder
+public class CacheDevConsoleRecorder {
+
+    public Handler<RoutingContext> clearCacheHandler() {
+        return new DevConsolePostHandler() {
+            @Override
+            protected void handlePost(RoutingContext event, MultiMap form) throws Exception {
+                String cacheName = form.get("name");
+                for (CaffeineCache cache : CaffeineCacheSupplier.allCaches()) {
+                    if (cache.getName().equals(cacheName)) {
+                        cache.invalidateAll();
+                        // redirect to the same page so we can make sure we see the updated results
+                        flashMessage(event, "Cache for " + cacheName + " cleared");
+                        return;
+                    }
+                }
```

Comment:
If this method is the only reason why `CacheManagerImpl.allCaches` was added, then I'd suggest to iterate over the caches names using `CacheManager.getCacheNames` and then retrieve the desired cache using `CacheManager.getCache`.

Reply:
Probably, yes.

Reply:
Yeah

Reply:
Rewrote this, is it better?

Reply:
Yes, thanks!